enos/ansible/roles/influx/tasks/main.yml

```diff
@@ -40,24 +46,20 @@
     delay: 2
     timeout: 120
 
-- name: Create the cadvisor database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE cadvisor'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the telegraf database
+  influxdb_database:
+    hostname: "localhost"
+    database_name: "telegraf"
+    state: present
 
-- name: Flush the annotation database if exists
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=DROP DATABASE IF EXISTS events'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
-
-- name: Create the annotation database
-  shell: "curl -s -XPOST http://{{ influx_vip }}:8086/query --data-urlencode 'q=CREATE DATABASE events'"
-  register: result
-  until: result.stdout == '{"results":[{}]}'
-  retries: 10
-  delay: 2
+- name: Create the telegraf database
```

Comment:
May I suggest: `name: Flush the annotation database (if exists)`

Reply:
sure